Tests/LeafKitTests/TagTests.swift

```diff
@@ -117,6 +117,21 @@ class TagTests: XCTestCase {
         """
         try XCTAssertEqual(render(template, ["core": ["Tanner", "Logan", "Gwynne", "Siemen", "Tim"]]), expected)
     }
+    
+    func testIsEmpty() throws {
```

Comment:
Could you add a check in the test to ensure the inverse also works - so `isEmpty` not returning true? Otherwise good to go!

Reply:
Of course. See the latest commit.